sepia-fog-images/config/definitions/sepia-fog-images.yml

```diff
@@ -32,6 +32,10 @@
           name: CMANSIBLEBRANCH
           default: "master"
           description: "Optionally define a different ceph-cm-ansible branch (useful for testing)"
+      - string:
+          name: PAUSEQUEUE
+          default: "true"
+          description: "Should the teuthology queue be paused?  Recapturing an existing OS image will cause running reimages to fail without pausing the queue.  The queue can remain unpaused when a new distro/version is being captured."
```

Comment:
it would be useful to add in the description what the default is

Reply:
Done!